tools/junitreport/.gitignore

```diff
@@ -0,0 +1 @@
+junitreport
```

Comment:
@soltysh why doesn't this correctly pick up the binary `junitreport` when I run `go build ./...` from this directory?